packages/form-builder/src/types/formSettings.ts

```diff
@@ -8,4 +8,6 @@ export type FormSettings = {
     registration: string;
     goalFormat: string;
     templateId: string;
+    primaryColor: string;
+    secondaryColor: string;
```

Comment:
Let's add the GiveWP branding colors as the defaults for these in `App.tsx` within initialState.  If these are empty the template is going to look very strange.

Reply:
Resolved by https://github.com/impress-org/givewp-next-gen/pull/87/commits/52496c8e4f37b35d767dc56f1e0191d13e007a02.